webapp/channels/src/components/post_view/message_attachments/message_attachment/__snapshots__/message_attachment.test.tsx.snap

```diff
@@ -29,12 +29,12 @@ exports[`components/post_view/MessageAttachment should call actions.doPostAction
         key="attachment__author-name"
         location="message_attachment"
       >
-        <Connect(ExternalImage)
+        <Connect(Component)
```

Comment:
I am not worried about this but i think this happened because of memo @larkox do you think adding display name is a good idea

Reply:
I want to shy away from shallow snapshot testing at some point. That is why I haven't given it more thought.

Also, I think this is an artifact of shallow rendering. Keep in mind that you are not passing the component to the connector, but the memo. So the shallow rendering has not gone deep enough there. In order for this to show the correct name I think we should do something like:
```
const memoizedComponent = React.memo(Component);
memoizedComponent.displayName = 'SomeName'
export default memoizedComponent;
```
Which is a lot of boilerplate to just make the shallow tests happy.

Reply:
Yes i agree on your point